src/pages/mixins/BrowsePageMixin.js

```diff
@@ -23,7 +23,7 @@ const BrowsePage = {
     },
   },
   mounted() {
-    if (this.query.q && !this.$store.state.images.length) {
```

Comment:
This was added to fix another bug. See https://github.com/creativecommons/cccatalog-frontend/pull/514

I think the fix this issue requires some revamping of the loading of images in the store and not a small change in the logic of this component. But that will require some research and refactoring

Reply:
Okay,  I would go ahead with the research and give updates on how I plan to solve the issue
Let me know if you’re fine with this.

Reply:
Okay, I would go ahead to research this and update you on how I plan to solve the issue.
Let me know if you’re okay with this.